src/views/Home.vue

```diff
@@ -433,6 +433,20 @@
                   >CI Tech Sensors</a
                 >
               </slide>
+              <slide class="has-text-centered">
+                <div class="has-image-centered customer-logo-wide">
+                  <figure class="image">
+                    <img src="../assets/emmi.png" alt="Emmi" />
+                  </figure>
+                  <br />
+                </div>
+                <a
+                  href="https://group.emmi.com/che/en"
```

Comment:
I would just link to `https://group.emmi.com` and let the user's browser decide upon the language.

Reply:
Agreed. Done.